jina/serve/networking.py

```diff
@@ -315,15 +330,23 @@ def __init__(
             self,
             address,
             channel,
+            deployment_name: str,
             metrics: _NetworkingMetrics,
             histograms: _NetworkingHistograms,
         ):
             self.address = address
             self.channel = channel
+            self.deployment_name = deployment_name
             self._metrics = metrics
             self._histograms = histograms
             self._initialized = False
 
+            if self._histograms:
```

Comment:
No need for an `else` statement?

Reply:
I guess the `self._histograms` is always checked before accessing the `self.stub_specific_labels`.

Reply:
Yeap. I think that's the better pattern rather than having a default None.

Reply:
That way it's obvious that self.stub_specific_labels is coupled to the existence of self._histograms.